src/_guides/language/language-tour.md

```diff
@@ -1041,9 +1041,9 @@ void main() {
 {% endcomment %}
```

Comment:
Globally we'll need to change "click the run button |>" to "click **Run**" (no image).

Reply:
This file has one occurrence of "click run". (You caught all the others in this page.)